### PR TITLE
[CH]Update Readme for clickhouse backend

### PR DESCRIPTION
**Pull request by lgbo-ustc**
## What changes were proposed in this pull request?

update readme for clickhouse backend. Add some usage instructions,

## How was this patch tested?

(Please explain how this patch was tested. E.g. unit tests, integration tests, manual tests)

(If this patch involves UI changes, please attach a screenshot; otherwise, remove this)

### CODE REVIEW

File: docs/ClickHouse.md [hunk 1]
```diff
@@ -242,6 +243,99 @@ bin/beeline -u jdbc:hive2://localhost:10000/ -n root
...
+  --master local[3] \
+  --files $hdfs_conf_file \
+  --driver-memory 10g \
+  --conf spark.serializer=org.apache.spark.serializer.JavaSerializer \
+  --conf spark.sql.sources.ignoreDataLocality=true \
+  --conf spark.default.parallelism=1 \
+  --conf spark.sql.shuffle.partitions=1 \
+  --conf spark.sql.files.minPartitionNum=1 \
+  --conf spark.sql.files.maxPartitionBytes=1073741824 \
+  --conf spark.sql.adaptive.enabled=false \
```
Comment (zzcclp) [inline docs/ClickHouse.md:291]: delete this line, use AQE as default.

**Review by zzcclp — APPROVED** [review]
LGTM

**Comment by zzcclp** [comment]
Please rebase .

**Comment by lgbo-ustc** [comment]
had rebased

**Comment by binmahone** [comment]
LGTM

**Pull request merged**